cmd/ocm/list/user/cmd.go

```diff
@@ -0,0 +1,152 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package user
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift-online/ocm-cli/pkg/arguments"
+	c "github.com/openshift-online/ocm-cli/pkg/cluster"
+	"github.com/openshift-online/ocm-cli/pkg/config"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+)
+
+var args struct {
+	clusterKey string
+	parameter  []string
+	header     []string
+	step       bool
+	columns    string
+	padding    int
+}
+
+// Cmd Constant:
+var Cmd = &cobra.Command{
+	Use:     "users --cluster=mycluster [flags]",
+	Aliases: []string{"user"},
+	Short:   "List cluster users",
+	Long:    "List administrative cluster users",
+	Args:    cobra.RangeArgs(0, 1),
+	RunE:    run,
+}
+
+func init() {
+	fs := Cmd.Flags()
+	fs.StringVarP(
+		&args.clusterKey,
+		"cluster",
+		"c",
+		"",
+		"Name or ID of the cluster to add the IdP to (required).",
+	)
+	//nolint:gosec
+	Cmd.MarkFlagRequired("cluster")
+	arguments.AddParameterFlag(fs, &args.parameter)
+	arguments.AddHeaderFlag(fs, &args.header)
+	fs.BoolVar(
+		&args.step,
+		"step",
+		false,
+		"Load pages one step at a time",
+	)
+	fs.StringVar(
+		&args.columns,
+		"columns",
+		"id, name, api.url, openshift_version, cloud_provider.id, region.id, state",
+		"Specify which columns to display separated by commas, path is based on Cluster struct",
```

Comment:
This seems wrong. Should be user columns?

Reply:
oops :)